api/utils/DocumentsHelper.js

```diff
@@ -0,0 +1,69 @@
+const castArray = require('./ArrayHelpers').castArray;
+
+const getAllDocuments = (medicalReport) => {
+  if (medicalReport.supportingDocuments) {
+    return medicalReport.supportingDocuments;
+  }
+  return false;
+}
+
+const getDocumentsByCondition = (medicalReport, conditionId) => {
+  if (medicalReport.supportingDocuments) {
+
+    const documents = medicalReport.supportingDocuments;
+    const conditionDocuments = [];
+
+    documents.forEach((document) => {
+      if (document.conditions.indexOf(conditionId) !== -1) {
+        conditionDocuments.push(document.name);
+      }
+    });
+
+    return conditionDocuments;
+  }
+  return false;
```

Comment:
Suggestion: Might be better to return `undefined` or `null` here.

Reply:
agreed